docs/admin/admission-controllers.md

```diff
@@ -238,25 +275,29 @@ three system reserved namespaces `default`, `kube-system`, `kube-public`.
 A `Namespace` deletion kicks off a sequence of operations that remove all objects (pods, services, etc.) in that
 namespace.  In order to enforce integrity of that process, we strongly recommend running this plug-in.
 
-### DefaultStorageClass
+### NodeRestriction
 
-This plug-in observes creation of `PersistentVolumeClaim` objects that do not request any specific storage class
-and automatically adds a default storage class to them.
-This way, users that do not request any special storage class do no need to care about them at all and they
-will get the default one.
+This plug-in limits the `Node` and `Pod` objects a kubelet can modify. In order to be limited by this admission plugin,
+kubelets must use credentials in the `system:nodes` group, with a username in the form `system:node:<nodeName>`. 
+Such kubelets will only be allowed to modify their own `Node` API object, and only modify `Pod` API objects that are bound to their node.
+Future versions may add additional restrictions to ensure kubelets have the minimal set of permissions required to operate correctly.
 
-This plug-in does not do anything when no default storage class is configured. When more than one storage
-class is marked as default, it rejects any creation of `PersistentVolumeClaim` with an error and administrator
-must revisit `StorageClass` objects and mark only one as default.
-This plugin ignores any `PersistentVolumeClaim` updates, it acts only on creation.
+### OwnerReferencesPermissionEnforcement
 
-See [persistent volume](/docs/user-guide/persistent-volumes) documentation about persistent volume claims and
-storage classes and how to mark a storage class as default.
+This plug-in protects the access to the `blockOwnerDeletion` property of an object
+so that only users with "delete" permission to the object can change it.
```

Comment:
This plug-in protects the access to `metadata.ownerReferences` of an object so that only users with "delete" permission to the object can change it.
This plug-in protects the access to `metadata.ownerReferences[x].blockOwnerDeletion` of an object, so that only users with "delete" permission to the ***owner***, i.e., the object referred by the `ownerReference[x]`, can change it.
the `blockOwnerDeletion` property of an object